packages/home-spa/README.md

```diff
@@ -0,0 +1,47 @@
+# Welcome to home-spa 👋
+![Version](https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000)
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/1-Platform/one-platform#readme)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/one-platform/graphs/commit-activity)
+
+> Home SPA for One Platform
+
+### 🏠 [Homepage](https://github.com/1-Platform/one-platform#readme)
+
+## Install
+
+```sh
+npm install
+```
+
+## Usage
+### Development server
+```sh
+npm run dev
+```
+
+### Build
+```sh
+npm run build
+```
+
+## Run tests
+
+```sh
+npm run build
+npm t
```

Comment:
Please use full forms for documentation purposes. You may specify aliases or short forms of commands as alternatives.

Reply:
![Screenshot from 2020-03-02 15-38-00](https://user-images.githubusercontent.com/12994292/75666499-ea5ef780-5c9b-11ea-8be1-a0a0825fe627.png)

Changed to full form